src/frontend/stores/component-data/component-data-store.ts

```diff
@@ -97,6 +97,45 @@ export class ComponentDataStore extends AbstractStore {
   }
 
   /**
+   * Get the nodes from root of the tree to given node.
+   * @param  {Object} node
+   */
+  private getPathToNode(node: any): any[] {
+    // The ID shows the path from root to given node.
+    let splitId: number[] = node.id.split('.');
+    let nodes: any[] = [];
+
+    // First index is for the component root:
+    let rootIndex: number = splitId.shift();
```

Comment:
@andrewlo I think that variable `rootIndex` is inferred to have the type of number by the compiler. You can omit an explicit type declaration.